.github/workflows/release.yml

```diff
@@ -105,12 +105,21 @@ jobs:
           CONTEXT=$(find . -name '*_srtool_output.json')
           SRTOOL() { <$(<<<$CONTEXT head -n1) jq -r .$1; }
           WASM() { <${JSON} jq -r ".runtimes.compressed.subwasm.$1"; }
+
+          # Copy the relevant parts of the changelog
+          .github/changelog-processor.py CHANGELOG.md --print-changelog-from-last-release > DRAFT
```

Comment:
I dont quite remember, but do we replace the `Unreleased` with the version number before calling this?  
Otherwise we have to replace it here.

Reply:
Yes. When a pr is merged to master that replaces `Unreleased` with a version, this entire machinery here is started.